go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,61 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"os/user"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the NativeMessage whitelists for root user.
+	Root string
+	// Home is the path of the NativeMessage whitelists for regular users.
+	Home string
+	// Overlay is an optional path to prefix all paths with. Useful for
+	// generating whitelists for Linux distributio packaging.
+	Overlay string
```

Comment:
It made sense to me because it's a path component (and that's what the rest of the struct is, just a bunch of path info).

It's not something that makes sense in all Install types, though. It only applies for the write-a-json-file type (*nix). On windows, it's a registry-based thing which works differently.

Also I figure since whitelistPath is a private type, it's not really leaking much.